docs/BesuProxyModel.md

```diff
@@ -1,33 +1,17 @@
 
 **Testing Avalon Proxy model with Hyperledger Besu**
-1.  Install ``truffle`` using the command
-    ```
-    sudo npm install -g truffle
-    ```
-    This requires Requires ``NodeJS v8.9.4 or later`` preinstalled.
-
-2.  Set the environment variable ``TCF_HOME`` to the Avalon root directory. Update ``no_proxy`` environment variable if you are behind a proxy
+1.  Set the environment variable ``TCF_HOME`` to the Avalon root directory. Update ``no_proxy`` environment variable if you are behind a proxy
```

Comment:
Mention what should we set no_proxy value.

Reply:
It's there already @Ram-srini